test/core.js

```diff
@@ -284,6 +268,25 @@ QUnit.test( "jQuery.holdReady (warn only)", function( assert ) {
 	} );
 } );
 
+QUnit.test( "jQuery.isFunction", function( assert ) {
+	assert.expect( 4 );
+
+	expectWarning( assert, "jQuery.isFunction", function() {
+		assert.equal( jQuery.isFunction( function(){} ), true, "function is function" );
+		assert.equal( jQuery.isFunction( {} ), false, "object not function" );
+		assert.equal( jQuery.isFunction( 1 ), false, "number not function" );
```

Comment:
Can you add a regexp test? We had to guard against that in the past.

Reply:
Yeah if I copy the full implementation I should pull in the full test suite too.